File: example1/src/Create.java
public class Create extends Element {
    public Create(double delay) {
        super(delay);
        super.setTnext(0.0); // імітація розпочнеться з події Create
    }

    @Override
    public void outAct() throws ExceptionInvalidTimeDelay {
        super.outAct();
        super.setTnext(super.getTcurr() + super.getDelay());
        super.getNextElement().inAct();
    }

    @Override
    public void inAct() throws ExceptionInvalidTimeDelay {

    }

    @Override
    public void doStatistics(double delta) {

    }
}

File: C#/Lab2/Create.cs
namespace Lab2;

public class Create : Element
{
    public Create(double delay = 1.0, string name = "",  string distribution = "exp") : base(delay, name, distribution)
    {
        Tnext = 0.0;
    }

    public override void OutAct()
    {
        base.OutAct();
        Tnext = Tcurr + GetDelay();
        NextElement?.InAct();
    }
}